packages/filesystem/src/node/node-filesystem.ts

```diff
@@ -259,7 +259,10 @@ export class FileSystemNode implements FileSystem {
         const _uri = new URI(uri);
         const stat = await this.doGetStat(_uri, 0);
         if (stat) {
-            throw FileSystemError.FileExists(uri, 'Error occurred while creating the directory.');
+            if (stat.isDirectory) {
+                return stat;
```

Comment:
@akosyakov as an API consumer, I would assume that when I create a directory: if it is already there then I am happy.

Either that or an additional `ensureFolder` API? wdyt?